opentelemetry-appender-tracing/examples/basic.rs

```diff
@@ -15,9 +18,19 @@ fn main() {
         )]))
         .with_simple_exporter(exporter)
         .build();
-    let layer = layer::OpenTelemetryTracingBridge::new(&provider);
-    tracing_subscriber::registry().with(layer).init();
-
-    error!(name: "my-event-name", target: "my-system", event_id = 20, user_name = "otel", user_email = "otel@opentelemetry.io", message = "This is an example message");
+    let env_filter = EnvFilter::from_default_env();
+    let otel_layer = layer::OpenTelemetryTracingBridge::new(&provider);
+    let internal_log_layer = fmt::Layer::default()
+        .with_writer(std::io::stdout) // Writes to stdout
+        .pretty()
+        .with_filter(filter_fn(|meta| meta.target().starts_with("opentelemetry"))) // Custom filter function
+        .with_filter(env_filter);
+    tracing_subscriber::registry()
+        .with(internal_log_layer)
```

Comment:
as the `internal_log_layer` can be added only after the `LoggerProvider` is created, any internal logs till that time won't be logged. This would be the case for logs and traces, and not metrics.